### DISCUSSION TITLE
Signalr integration

### DISCUSSION THREAD

**Discussion by HolgerHuckfeldt** [elsa-workflows/elsa-core#1370] [Ideas]
Has anybody already thought about integrating elsa with SignalR?
It would be nice to have the possibility to signal changes to some businessobjects immediatly to the clients.

It should not be to hard to code a common hub and to establish communication between elsa core and clients.

Are there plans to do so? Should I do it?

**Comment by sfmskywalker** [comment]
Having SignalR activities to send information to the clients would be awesome. I don't think anyone is working on that, so please feel free to create an issue that outlines what you have in mind.

**Reply by HolgerHuckfeldt** [reply]
Ok, is ongoing... :-)
